README.md

```diff
@@ -57,13 +100,15 @@ Run `ng g component my-component --project=my-app` to generate a new component.
 
 Run `ng build my-app` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
```

Comment:
```suggestion
To run the project on a development server, run 
`ng build my-app

The build artifacts are stored in the `dist/` directory. 

To build the app to production, use the `--prod` flag.
```
The title for this section appears to be for the development server. Should this be a separate title for production?
Perhaps you can have this as a single title: 
Building your app

and the different commands in for the different build environments.